src/icons/MinusCircleIcon.tsx

```diff
@@ -1,23 +1,35 @@
-import React from 'react';
+/* eslint-disable react/display-name */
+import * as React from 'react';
 
-import { SvgIcon } from '@material-ui/core';
+import { styled, SvgIconConfig } from 'stitches.config';
 
-export const MinusCircleIcon = (props: any) => (
-  <SvgIcon {...props}>
-    <path
-      d="M12 20.3333C16.6024 20.3333 20.3333 16.6024 20.3333 12C20.3333 7.39763 16.6024 3.66667 12 3.66667C7.39763 3.66667 3.66667 7.39763 3.66667 12C3.66667 16.6024 7.39763 20.3333 12 20.3333Z"
-      stroke="currentColor"
-      fill="none"
-      strokeWidth="1.5"
-      strokeLinecap="round"
-      strokeLinejoin="round"
-    />
-    <path
-      d="M8.66667 12H15.3333"
-      stroke="currentColor"
-      strokeWidth="1.5"
-      strokeLinecap="round"
-      strokeLinejoin="round"
-    />
-  </SvgIcon>
+import { SvgIcon } from './SvgIcon';
+
+import { IconProps } from 'types';
+
+export const MinusCircleIcon = styled(
+  ({ color = 'currentColor', ...props }: IconProps) => {
+    return (
+      <SvgIcon {...props}>
```

Comment:
I still think it is possible to remove a couple lines of boiler plate, I don't want to ask you to spend so much time on this..

whatif it was like: 
```
export const MinusCircleIcon = (props) =>
  <SvgIcon {...props}>
    ({ color }) => (
       <path d="M12 10 2 1" stroke={color} />
     )
   </SvgIcon>
   ```
   I think needing color is why you kept those parts of the interface out?

Reply:
I need to wrap into the `styled`